.yamllint.yaml

```diff
@@ -0,0 +1,31 @@
+---
+extends: default
+
+rules:
+  # 180 chars should be enough, but don't fail if a line is longer.
+  line-length:
+    max: 180
```

Comment:
do we also need `allow-non-breakable-inline-mappings`? (https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.line_length)

Reply:
Don't know, it's a choice, I have never used it until now, but it can be added.